server/src/main/java/org/candlepin/pinsetter/tasks/UniqueByEntityJob.java

```diff
@@ -25,7 +25,7 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * UniqueByOwnerJob can by extended by jobs that should not be run concurrently
+ * UniqueByEntityJob can by extended by jobs that should not be run concurrently
  * per owner. Additional scheduled jobs will be ignored if a similar job exists
```

Comment:
This is my bad, should have cleaned up myself when I implemented it, :-1: 
but perhaps we could change "per owner" to "per target entity ( owner , consumer, etc )"